PyInstaller/hooks/hook-PySide2.py

```diff
@@ -10,13 +10,19 @@
 
 from PyInstaller.utils.hooks import collect_system_data_files
 from PyInstaller.utils.hooks.qt import pyside2_library_info, get_qt_binaries
+import PyInstaller.compat as compat
```

Comment:
Why not just `from PyInstaller.compat import is_win`?

Reply:
Changed to the narrower import